src/database/models/base.py

```
@@ -0,0 +1,7 @@
+from sqlalchemy.orm import DeclarativeBase
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)